projects/packages/masterbar/src/admin-menu/class-admin-menu.php

```diff
@@ -114,14 +114,7 @@ public function should_disable_links_manager() {
 	 * Adds My Home menu.
 	 */
 	public function add_my_home_menu() {
-
-		if ( self::DEFAULT_VIEW !== $this->get_preferred_view( 'index.php' ) ) {
-			// @phan-suppress-next-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539.
-			add_menu_page( __( 'My Home', 'jetpack-masterbar' ), __( 'My Home', 'jetpack-masterbar' ), 'read', 'https://wordpress.com/home/' . $this->domain, null, 'dashicons-admin-home', 1.5 );
-			return;
-		}
-
-		$this->update_menu( 'index.php', 'https://wordpress.com/home/' . $this->domain, __( 'My Home', 'jetpack-masterbar' ), 'read', 'dashicons-admin-home' );
+		// TODO: Kept to avoid mid-deploy fatals. Remove this in a follow-up.
```

Comment:
We don't need to care about mid-deploy fatals for Jetpack since the deployment is done with the moon-sun strategy (blue green deployment)